### DISCUSSION TITLE
Wineskin goes unresponsive after program install

### DISCUSSION THREAD

**Discussion by andystevensname** [The-Wineskin-Project/WineskinServer#275] [Q&A] [answered]
First, I appreciate all the hard work that's been done with this project, thank you!

I'm trying to troubleshoot an install on macOS Monterrey 12.4. To give some background: I downloaded Wineskin through Homebrew and successfully managed to install the Windows program I'd like to use, Magic: the Gathering Online, after installing some things through Winetricks (corefonts, render=gdi, win7, dotnet472, gdiplus, gdiplus=native). I had to use the WS11WineCX64Bit20.0.4 engine, as for some reason .NET 4.7.2 kept crashing wineserver on installation with the WS11WineCX84Bit21.2.0 engine.

But, this all worked after the second or third try of tinkering. However, Magic's setup.exe is an interesting one in that the setup.exe, in addition to being the install package, is also the launcher for the program. I had the setup.exe on my desktop. I tried moving it to various places on my mac to be a bit more organized, and the .app failed to open. Opening the Wineskin app inside the MTGO.app package contents revealed that a .bat file had been installed and that .bat was basically just a call for wine on the Z: drive to the setup.exe, so it would make sense that the MTGO.app would no longer work if I moved that setup.exe

No worries, I'll start over and move things to a more permanent location. This time everything worked again, but on logging in the game just kept spinning on the loading screen. In retrospect, it might have been something to do with the MTGO servers, but I thought that perhaps there was a problem with the install. Okay, delete the MTGO.app and start over.

It was around this time that no matter what I did, I couldn't get the setup.exe to move past it's initial setup and trigger the game to load. I've successfully "installed" the game, as in the setup.exe downloads about half a gig worth of files and I can see it in the uninstall menu under Advanced Options, but while it previously had triggered the actual game to start, the launcher closes, a win64-preloader process starts, and nothing else happens.

I've removed everything I could possibly think of and reinstalled Wineskin Winery again in the hopes that maybe there was a configuration file missing or in a broken state, but the problem persists. Here's what I removed:

    /Users/<username>/Applications/Wineksin/MTGO.app
    /Users/<username>/Library/Applications Support/Wineskin
    /Users/<username>/Library/Caches/Homebrew/downloads/ < I think I got the Wineskin files
    /Users/<username>/.cache/winetricks

And of course I uninstalled Wineskin via Homebrew.

I've re-run everything with new blank wrappers at least a dozen times, restarted my computer a bunch, removing wine64-preloader and win32on64-preloader system processes each time in case something was sticking around that shouldn't be.

I've also tried copying the setup.exe into the C: folder. That prevented the .bat file from being generated. On launching the .app with this configuration a prompt shows up that checks to see if there's an update needed for the program (which is good), but then it closes, a wine64-preloader window appears, and then nothing else happens.

So I guess my big question is, is there anything else I can "reset" to go back to a clean working install of Wineskin Winery? Is there any log or something I could provide to help in troubleshooting? This might be a tough nut to crack, and I know time is precious, so any pointers or things I can track down are greatly appreciated.

Thank you for taking the time to read this.

**Comment by Gcenx** [comment]
Is this a free game like MTGA?

**Reply by andystevensname** [reply]
This game is free in that you don't have to purchase it [[download here](https://magic.wizards.com/en/mtgo#download-now)], but it's not free like MTGA in that you can play it without purchasing any MTG cards. MTGO is the precursor of MTGA, it's been out for at least a decade.

**Reply by andystevensname** [reply]
@Gcenx Did you have any other ideas of what I might do to troubleshoot this issue? If not, no worries, as a last-ditch effort I could reinstall macOS.

**Comment by emendelson** [comment, marked as answer]
You could delete the Wineskin Winery app and the Wineskin folder in ~/Library/Application Support. That should get rid of every trace of it.

**Reply by andystevensname** [reply]
Thanks @emendelson. I ripped everything out again, as I had done before, but it worked this time! Success!
